src/pages/Note.js

```diff
@@ -1,39 +1,131 @@
 /** @jsx jsx */
 
+import { NavLink } from "react-router-dom";
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useSpring, animated } from "react-spring";
 import { css, jsx } from "@emotion/core";
+import { useSelector } from "react-redux";
 
 import { useQuery } from '@apollo/react-hooks';
 import gql from "graphql-tag";
 
+import { getTimeAgoString } from "../util";
 
-const GETNOTE = gql`
-    query  {
-        notes {
-        id
-        title
-        titleId
-        createdBy{
-            username
+const styles = css`
+    .noteDiv {
```

Comment:
Can you rename "noteDiv" to "note-container"

Reply:
<h2><i>changes<br/><br/>made</i></h2>